examples/messaging/pub-sub/dotnet/Main.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Text;
+using NATS.Client;
+
+using NATS.Client;
+
+string natsURL = Environment.GetEnvironmentVariable("NATS_URL");
+if (natsURL == null)
+{
+    natsURL = "nats://127.0.0.1:4222";
+}
+
+// Create a new connection factory to create a connection.
+Options opts = ConnectionFactory.GetDefaultOptions();
+opts.Url = natsURL;
+
+// Creates a live connection to the default
+// NATS Server running locally
+ConnectionFactory cf = new ConnectionFactory();
+IConnection c = cf.CreateConnection(opts);
+
+// Here are some of the accessible properties from
+// the Msg object:
+//   Msg.Data;
+//   Msg.Reply;
+//   Msg.Subject;
+//   Msg.Header;
+//   Msg.MetaData;
+
+// Setup an event handler to process incoming messages.
+// An anonymous delegate function is used for brevity.
+EventHandler<MsgHandlerEventArgs> handler = (sender, args) =>
+{
+    // print the message
+    Msg m = args.Message;
+    string text = Encoding.UTF8.GetString(m.Data);
+    Console.WriteLine($"Async handler received the message '{text}' from subject '{m.Subject}'");
+};
+
+// Subscriptions will only receive messages that are
+// published after the subscription is made, so for this
+// example, we'll publish a message before we are subscribed
+// which will not be received.
+c.Publish("greet.joe", Encoding.UTF8.GetBytes("hello joe 1"));
+
+// The simple way to create an asynchronous subscriber
+// is to simply pass the handler in. Messages will start
+// arriving immediately. We are subscribing to anything that
+// matches the greet.* pattern. You will see that this
```

Comment:
Rendered comments support markdown, so slapping some backticks on it.

```suggestion
// matches the `greet.*` pattern. You will see that this
```

Reply:
fixed